terraform/rancher/project/keycloak.tf

```diff
@@ -0,0 +1,177 @@
+locals {
+  kc_admin_user_name  = "admin"
+  kc_target_http_port = "8080"
+}
+
+resource "helm_release" "keycloak" {
+  count      = (var.eureka ? 1 : 0)
+  chart      = "keycloak"
+  depends_on = [rancher2_secret.keycloak-credentials, helm_release.postgresql]
+  name       = "keycloak-${var.rancher_project_name}"
+  namespace  = rancher2_namespace.this.id
+  version    = "21.0.4"
+  repository = "https://repository.folio.org/repository/helm-bitnami-proxy"
+  values = [
+    <<-EOF
+image:
+  registry: 732722833398.dkr.ecr.us-west-2.amazonaws.com
+  repository: folio-keycloak
+  tag: latest
+  pullPolicy: Always
+  debug: true
```

Comment:
Do we need debug option for permanent solution?

Reply:
This "Debug" option is used for BITNAMI docker image and init code inside.
Here we can disable it.